knimin/lib/data_access.py

```diff
@@ -3315,6 +3215,28 @@ def get_plate_types(self):
             TRN.add(sql)
             return [dict(x) for x in TRN.execute_fetchindex()]
 
+    def read_plate_type(self, plate_type_id):
```

Comment:
The question is: do we have an actual string for plate_type_id or no? If yes, should we return that string vs. the id?